### PR TITLE
[CFM-558] Fix: Add 'news' to group content notification types, and also 'article' for comments.

### PR DESCRIPTION
**Pull request by anvmn**
https://issuetracker.amplexor.com/jira/browse/CFM-558

### CODE REVIEW

File: project/profiles/capacity4more/modules/c4m/og/c4m_og/c4m_og.module [hunk 1]
```diff
@@ -1631,7 +1631,17 @@ function c4m_og_get_user_administered_groups_count($uid, $node_type) {
  *   Notification types.
  */
 function c4m_og_get_group_content_notification_types() {
-  return array('discussion', 'document', 'event', 'wiki_page');
+  return array('discussion', 'document', 'event', 'wiki_page', 'news');
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/og/c4m_og/c4m_og.module]: @anvmn 
be aware that the `news` content type is only for projects and `discussion` is only for groups.

Comment (anvmn) [inline project/profiles/capacity4more/modules/c4m/og/c4m_og/c4m_og.module]: Don't see any problems that might be caused...

File: project/profiles/capacity4more/modules/c4m/og/c4m_og/c4m_og.module [hunk 2]
```diff
@@ -1631,7 +1631,20 @@ function c4m_og_get_user_administered_groups_count($uid, $node_type) {
...
+}
+
+/**
+ * Returns an array of content types that require notification om comment.
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/og/c4m_og/c4m_og.module]: om => on

File: project/profiles/capacity4more/modules/c4m/og/c4m_og/c4m_og.module [hunk 3]
```diff
@@ -1631,7 +1631,20 @@ function c4m_og_get_user_administered_groups_count($uid, $node_type) {
...
+ *
+ * @return array
+ *   Notification types.
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/og/c4m_og/c4m_og.module]: => Types require notifications for comments

**Pull request merged**